src/server/main_service.cc

```diff
@@ -665,6 +665,7 @@ void Service::Init(util::AcceptServer* acceptor, std::vector<facade::Listener*>
 
   shard_set->Init(shard_num, !opts.disable_time_update);
 
+  acl_family_.Init(listeners);
```

Comment:
do we enforce ACL on all listeners? I think we should only do it for the main listener (and not uds and admin).

Reply:
fixed, you are right.